### PR TITLE
Update start-translating-openedx.rst

### PR DESCRIPTION
**Pull request by jesgreco**
- Changing grammar structure to be consistent with the bullet list
- Spelling edit -> Capital letter

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @jesgreco!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.


:warning: We can't start reviewing your pull request until you've submitted a     [signed contributor agreement](https://openedx.org/cla)     or indicated your institutional affiliation. Please see the [CONTRIBUTING](https://github.com/openedx/.github/blob/master/CONTRIBUTING.md)     file for more information.     If you've signed an agreement in the past, you may need to re-sign.  See     [The New Home of the Open edX Codebase](https://open.edx.org/blog/the-new-home-of-the-open-edx-codebase/)     for details.

    
Once you've signed the CLA, please allow 1 business day for it to be processed.  After this time, you can re-run the     CLA check by adding a comment here that you have signed it.  If the problem persists, you can tag the `@openedx/cla-problems` team in a     comment on your PR for further assistance.

**Comment by mphilbrick211** [comment]
Hi @jesgreco! Thanks for this contribution! Please let me know if you have any question regarding submitting a [CLA form](https://openedx.atlassian.net/wiki/spaces/COMM/pages/941457737/How+to+start+contributing+to+the+Open+edX+code+base).

File: source/translators/quickstarts/start-translating-openedx.rst [hunk 1]
```diff
@@ -21,12 +21,12 @@ This tutorial assumes the following:
...
 * You have joined the Transifex `edx-platform project`_ with the language(s) you wish
   to translate to
-* Fluency in both English and the language(s) you are translating to. You should
+* You have to be fluent in both English and the language(s) you are translating to. You should
   have deep knowledge of English to be capable of translating phrases from the user
   interface and  documentation. You should become a translator only if you feel
   you are capable of translating in both English and your target language. Open edX is
   written in English, so all translations are done from English to another language
-* You have joined the Open edX Slack or Discuss. Links to join are available at the
+* You have joined the Open edX Slack or discuss. Links to join are available at the
```
Comment (feanil) [inline source/translators/quickstarts/start-translating-openedx.rst:29]:
```suggestion
* You have joined the Open edX Slack and Discourse (https://discuss.openedx.org) . Links to join are available at the
```

**Review by feanil — COMMENTED** [review]
Let's make this a bit more useful since not everyone might know what we mean by `discuss`.

**Comment by jesgreco** [comment]
I suggest deleting the word "discuss" since the communication is made by the Slack channel.

**Comment by feanil** [comment]
I think we want to make it clear that you can join both Discuss and slack. I know many comms happen in slack but I think both should be supported.

**Comment by jesgreco** [comment]
@feanil I've just committed your suggestion! Thanks! For me, it was not clear at the beginning, but now, yes!

**Review by feanil — APPROVED** [review]

**Comment by openedx-webhooks** [comment]
@jesgreco 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/docs.openedx.org&entry.867055334=https://github.com/openedx/docs.openedx.org/pull/418&entry.1484655318=https://github.com/jesgreco&entry.752974735=2023-10-27+13:42&entry.1917517419=2023-11-07+15:16&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**